charts/kueue/values.yaml

```diff
@@ -151,11 +151,16 @@ webhookService:
       targetPort: 9443
   type: ClusterIP
 
-# kueue-viz dashboard
-enableKueueViz: false
-
 metrics:
   prometheusNamespace: monitoring
   serviceMonitor:
     tlsConfig:
       insecureSkipVerify: true
+
+# kueue-viz dashboard
+enableKueueViz: false
+kueueViz:
+  backend:
+    image: "us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueue-viz-backend:main-latest"
```

Comment:
Why not `main` tag?

Reply:
afaik, there is no `main` tag that is getting built. The only one I have seen is `main-latest`

Reply:
I suggest that we have the same policy for tagging in kueueviz as for the main kueue. Otherwise it will be hard to understand. In main kueue we don't have latest at all. Please follow the same tagging strategy

Reply:
What do I replace? I took the values that were already default in the helm chart.

Reply:
https://github.com/kubernetes-sigs/kueue/pull/4748 to be fixed here